src/itensor.jl

```diff
@@ -271,25 +274,25 @@ ITensor(x::RealOrComplex{Int}, is...) = ITensor(float(x), is...)
 
 # TODO: Deprecated!
 """
-    emptyITensor([::Type{ElT} = NDTensors.EmptyNumber, ]inds)
-    emptyITensor([::Type{ElT} = NDTensors.EmptyNumber, ]inds::Index...)
+    emptyITensor([::Type{ElT} = NDTensors.default_eltype(), ]inds)
+    emptyITensor([::Type{ElT} = NDTensors.default_eltype(), ]inds::Index...)
 
-Construct an ITensor with storage type `NDTensors.EmptyStorage`, indices `inds`, and element type `ElT`. If the element type is not specified, it defaults to `NDTensors.EmptyNumber`, which represents a number type that can take on any value (for example, the type of the first value it is set to).
+Construct an ITensor with storage type `NDTensors.EmptyStorage`, indices `inds`, and element type `ElT`. If the element type is not specified, it defaults to `NDTensors.default_eltype()`, which represents a number type that can take on any value (for example, the type of the first value it is set to).
 """
 function emptyITensor(::Type{ElT}, is::Indices) where {ElT<:Number}
-  return itensor(EmptyTensor(ElT, is))
+  return itensor(NDTensors.Zeros{ElT,1,NDTensors.default_datatype(ElT)}(is), is)
 end
 
 function emptyITensor(::Type{ElT}, is...) where {ElT<:Number}
   return emptyITensor(ElT, indices(is...))
 end
 
-emptyITensor(is::Indices) = emptyITensor(EmptyNumber, is)
+emptyITensor(is::Indices) = emptyITensor(NDTensors.default_eltype(), is)
 
-emptyITensor(is...) = emptyITensor(EmptyNumber, indices(is...))
+emptyITensor(is...) = emptyITensor(NDTensors.default_eltype(), indices(is...))
 
-function emptyITensor(::Type{ElT}=EmptyNumber) where {ElT<:Number}
-  return itensor(EmptyTensor(ElT, ()))
+function emptyITensor(::Type{ElT}=NDTensors.default_eltype()) where {ElT<:Number}
+  return itensor(NDTensors.Zeros{ElT,1,NDTensors.default_datatype(ElT)}(()), ())
```

Comment:
Seems like now would be a good time to deprecate these. Probably we can just remove these definitions and put:
```julia
@deprecate emptyITensor(args...; kwargs...) ITensor(args...; kwargs...)
```
in `deprecated.jl` since it should just be doing the same thing as the default `ITensor(...)` constructor.